src/util/metric.rs

```diff
@@ -0,0 +1,76 @@
+// Copyright 2016 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use log::LogLevel;
+use cadence::prelude::{Counted, Timed, Gauged, Metered};
```

Comment:
It wont compile without these.
`Counted, Timed, Gauged, Metered` are triats that contain some method signatures used below.

eg: `Time` => https://github.com/pingcap/tikv/pull/577/files#diff-fa8093341d2e5ce6a6fb6bb4fbb1affeR50